test/e2e/framework/assertions.go

```diff
@@ -252,12 +223,10 @@ func (f *Framework) GetOperatorMetrics(t *testing.T) []byte {
 // AssertNoReconcileErrors asserts that there are no reconcilation errors
 func (f *Framework) AssertNoReconcileErrors(t *testing.T) {
 	metrics := f.GetOperatorMetrics(t)
-	assertPromQL(t, metrics,
-		`controller_runtime_reconcile_errors_total`,
+	assertSamples(t, metrics,
 		map[string]float64{
-			`{__name__="controller_runtime_reconcile_errors_total", controller="grafana-operator"}`: 0,
```

Comment:
I assume we're removing this as a part of the change as grafana-operator is no longer around in p-o